src/osal/lv_freertos.h

```diff
@@ -61,11 +61,12 @@ typedef struct {
 } lv_mutex_t;
 
 typedef struct {
+    BaseType_t
+    xIsInitialized;                       /**< Set to pdTRUE if this condition variable is initialized, pdFALSE otherwise. */
 #if USE_FREERTOS_TASK_NOTIFY
     TaskHandle_t xTaskToNotify;
+    BaseType_t xSignalSent;
```

Comment:
No need for `xSignalSent`, so it can be removed, correct? We make use of the `xIsInitialized` now.

Reply:
Not currently, but I will can make that variable common between both implementations :+1: